docs/guides/designing-the-delta-format.md

```diff
@@ -294,9 +294,9 @@ var change = [
 ]
 ```
 
-Since every character is described, explicit indexes and lengths are no longer necessary. This makes out of order indexes and overlapping ranges impossible to express.
+Since every character is described, explicit indexes and lengths are no longer necessary. This makes indexes out of order and overlapping ranges impossible to express.
```

Comment:
The original is hard to read. Will think of a better way to phrase this and make an update.